### PR TITLE
Make manner of collision editable on temp records

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/7741

## Testing
**URL to test:** 
https://deploy-preview-1317--atd-vze-staging.netlify.app/#/

**Steps to test:**
1. Go to a non-temp crash, the Manner of collision field should not be editable.
2. Go to a temp crash (can find them on the Create crash record tab), you should be able to edit and change the Manner of collision field.

---
#### Ship list
- [ ] Check migrations for any conflicts with latest migrations in master branch
- [ ] Code reviewed
- [x] Product manager approved

### CODE REVIEW

File: atd-vze/src/views/Crashes/crashDataMap.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ export const createCrashDataMap = isTempRecord => {
         },
         fhe_collsn_id: {
           label: "Manner of Collision ID",
-          editable: false,
+          editable: isTempRecord,
```
Comment (roseeichelmann) [inline atd-vze/src/views/Crashes/crashDataMap.js:48]: Cool to be reusing this code from https://github.com/cityofaustin/atd-vz-data/pull/1238!

**Review by mddilley — APPROVED** [review]
Tested on the deploy preview, and this works perfectly! 🚢 🚀 🙌

**Review by chiaberry — APPROVED** [review]
I can update temp records, cannot update the other ones.

🚢

**Review by patrickm02L — APPROVED** [review]
Worked like a charm! 💎

**Pull request merged**